docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -97,6 +97,30 @@ requirements file in the format documented with :ref:`collection_requirements_fi
 
    ansible-galaxy collection download -r requirements.yml
 
+A source collection directory can also be downloaded. The collection will be built with the mandatory ``galaxy.yml`` file.
```

Comment:
Wait, is `galaxy.yml` mandatory? If so, then any collection with a MANIFEST.json will have both? What happens if the metadata in them differs?

Reply:
Yes, it's mandatory to ``download`` directories (unrelated of the code changes in this PR). It's needed to build the tarfile for the download directory. If the ``galaxy.yml`` isn't found the build step fails. Without the code changes in this PR, both the `galaxy.yml` and a `MANIFEST.json` could both be present (they *shouldn't* be), and yes, the metadata could conflict and lead to unexpected behavior because the ``galaxy.yml`` would only be used initially.

Reply:
ah, thanks for the clarification